pishutdown.py

```diff
@@ -23,14 +27,17 @@ def buttonStateChanged(pin):
 
     if not (GPIO.input(pin)):
         # button is down
-        buttonPressedTime = datetime.now()
+        if buttonPressedTime is None:
```

Comment:
And anyway, thanks for sharing the code. It works like a charm on my Raspberry Pi, now. :-)